com/uber/kafkaSpraynozzle/stats/StatsDReporter.java

```diff
@@ -0,0 +1,61 @@
+package com.uber.kafkaSpraynozzle.stats;
+
+import com.timgroup.statsd.NonBlockingStatsDClient;
+import com.timgroup.statsd.StatsDClient;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * Statsd StatsReporter implementation
+ *
+ * To initialize this class you need to provide it the path to a .properties file containing:
```

Comment:
haha, that is very true :-P

i initially went this approach, it felt a little funny passing json in via the command line, but i'm not opposed to it.  Let me put that code back in so you can take a look.